### DISCUSSION TITLE
Popup says "✅ Incomplete" for a successful subscription transaction

### DISCUSSION THREAD

**Discussion by StartupYard2** [Glench/ExtPay#204] [General]
I just did a real world subscription transaction. It worked but the popup says "Incomplete"
<img width="487" alt="Screenshot 2024-04-05 at 16 34 12" src="https://github.com/Glench/ExtPay/assets/68431823/c1b4cc88-1e86-4286-8619-73118420fa50">

Is it me?

**Comment by burkybang** [comment]
One of my users reported this today too. I quickly emailed Glen about it. He recommended uninstalling the extension, reinstalling it, and then subscribing again.

It definitely seems like an issue though.

**Reply by StartupYard2** [reply]
Yes when I uninstalled and reinstalled, it then said "Active" as expected. When it happened it was on a fresh install though.

**Comment by Glench** [comment]
Hey, would you mind sending me the email or api key (from the ExtensionPay url) so I can look into it? And did your card have any issues when you were paying? Or did you use 2-factor auth with your card?

**Comment by Glench** [comment]
I believe this is now fixed. It seemed to be a rare caching issue for some users have they both logged in and had an expired / repaid subscription.
